app/assets/javascripts/initializePage.js

```diff
@@ -11,8 +11,8 @@
   initializeVideoPlayback, initializeDrawerSliders,
   initializeHeroBannerClose, initializeOnboardingTaskCard, initScrolling,
   nextPage:writable, fetching:writable, done:writable, adClicked:writable,
-  initializePaymentPointers, initializeSpecialNavigationFunctionality, initializeBroadcast,
-  initializeDateHelpers
+  initializePaymentPointers, initializeBroadcast, initializeDateHelpers,
```

Comment:
I like that you got rid of `initializeSpecialNavigationFunctionality, initializeBroadcast` :)